### ISSUE TITLE
Use lowdb instead of sqlite3

### ISSUE THREAD

**Issue by k1r0s** [bleenco/abstruse#414]
Due to well known issues with sqlite3 on node, I swear if you guys have any concerns on this and, if you will, use another persistance service rather than sqlite3. 

[lowdb](https://github.com/typicode/lowdb) is quite small, standalone, like sqlite3, easy to use and very lightweight. 

What are your thoughts about?

related: https://github.com/bleenco/abstruse/issues/413

**Comment by k1r0s** [comment]
This is a discussion rather than an issue. I wanted to pointed out that sqlite is not well supported (IMHO) and has frequent problems. Overall using an orm with a sql database for a project that aims to be lightweight (other products like mysql require a service to be running and are way powerful than is needed).

On the other hand, using `lowdb` (or anyother similar dependency) will reduce the amount of work that someone has to do in order to use this project https://github.com/bleenco/abstruse/blob/master/docs/DEPENDENCIES.md

**Comment by k1r0s** [comment]
I've been using `lowdb` for a while and is quite powerful, simple and reliable. I guess I can file a PR on this when I have some time if you're interested

**Comment by k1r0s** [comment]
I think im going to do so anyways since I need this up and running on my `odroid`.

**Comment by jkuri** [comment]
are you sure that `sqlite3` doesn't work on `odroid` because I have `odroid xu4` which has a bit different arch but I installed sqlite3 without any problems. Maybe you should compile sqlite3 from source if its not listed under deb packages but I am pretty sure it should work, it is written in C and it has no dependencies.

**Comment by k1r0s** [comment]
I can do it, yes. But is a messy process, thanks!

**Comment by jkuri** [comment]
give me a minute, I'll check on this.

**Comment by jkuri** [comment]
```
 ~/Desktop/sqlite-aarch64/ docker run -it --rm arm64v8/alpine:3.8
/ # apk add --no-cache sqlite
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/main/aarch64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/community/aarch64/APKINDEX.tar.gz
(1/5) Installing ncurses-terminfo-base (6.1_p20180818-r1)
(2/5) Installing ncurses-terminfo (6.1_p20180818-r1)
(3/5) Installing ncurses-libs (6.1_p20180818-r1)
(4/5) Installing readline (7.0.003-r0)
(5/5) Installing sqlite (3.24.0-r0)
Executing busybox-1.28.4-r1.trigger
OK: 13 MiB in 18 packages
```
it should work right off, I also checked on `aarch64v8/ubuntu:bionic` image and there is available package `sqlite3`. You don't need to compile it from source in any case.

**Comment by k1r0s** [comment]
so this issue can be closed ;) thanks @jkuri, really appreciate it

**Comment by jkuri** [comment]
no problem man. if you have any good ideas that should be included on v2.0.0 branch you can join on slack and we can discuss about it when you have time.
